### ISSUE TITLE
[BUG] Chrome Checker unable to fully scan in incognito mode

### ISSUE THREAD

**Issue by Fullchee** [IBMa/equal-access#187]
## What tool(s) are you using?
https://chrome.google.com/webstore/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp
IBM Equal Access Accessibility Checker
## Detailed description

> Platform: MacOS Catalina
> Chrome: 84.0.4147.105

## Steps to reproduce the issue

1. Download the Extension
2. Allow the extension in incognito mode (in chrome://extensions)
3. Scan any website (i.e. google.ca)

It works as expected when not in incognito mode.

**Comment by zhenwan** [comment]
Hi, Thank you for reporting this issue. 
I tried Accessibility Checker with incognito mode on, I can not re-produce this issue. 

Can you provide more information to help me debug this issue:
1. Please send me a screen capture of the Details page
2. Click "background page" link, and copy and paste the logs in the console.
![image](https://user-images.githubusercontent.com/14806866/90042822-d0850000-dc90-11ea-8f67-53873d46be08.png)

**Comment by Fullchee** [comment]
```
Main._createAppUI: 17.302978515625ms
main.js:31 Main._showAppUI: 22.3740234375ms
main.js:31 Main._initializeTarget: 7.01904296875ms
main.js:31 Main._lateInitialization: 5.4560546875ms
```

![one](https://user-images.githubusercontent.com/11246258/90049371-b6541d80-dca2-11ea-8756-af21c2b97d6a.jpg)

![image](https://user-images.githubusercontent.com/11246258/90049285-96245e80-dca2-11ea-8f4f-dc8cf4031b2b.png)

**Comment by Fullchee** [comment]
I removed the extension and readded it but I'm getting the same error.

**Comment by Fullchee** [comment]
I'm also able to reproduce it on a Windows 10 machine as well.

**Comment by zhenwan** [comment]
Thanks, Fullchee, 
I can re-produce this issue now.

**Comment by zhenwan** [comment]
Hi Fullchee,

This issue is fixed in Pull Request #199. 

This is Checker's output from Incognito mode:
![image](https://user-images.githubusercontent.com/14806866/91078094-c0154380-e607-11ea-9931-938e1e901590.png)
